README.md

```diff
@@ -98,20 +98,32 @@ should be included, feel free to make the changes and submit a *pull-request*.
 
 To build the docs in your local environment, from the project directory:
 
-    tox -e docs-html
+```bash
+tox -e docs
+```
 
 ## Installation
 
 To install Flask-Admin, simply:
 
-    pip install flask-admin
+```shell
+pip install flask-admin
+```
 
 Or alternatively, you can download the repository and install manually
 by doing:
 
-    git clone https://github.com/pallets-eco/flask-admin.git
-    cd flask-admin
-    pip install .
+```shell
+git clone https://github.com/pallets-eco/flask-admin.git
+cd flask-admin
+pip install .
+```
+
+If you are contributing a bug fix or a new feature, you can install the app in editable mode:
+
+```shell
+pip install -e .
```

Comment:
Or should we do ".[all]"?

Reply:
I think `.[all]` is a good thought (and it's what I tend to do) 👍